cypress.json

```diff
@@ -1,4 +1,4 @@
 {
   "video": false,
-  "baseUrl": "http://localhost"
+  "baseUrl": "http://127.0.0.1"
```

Comment:
This drastically reduces the test's runtime, looks like DNS resolving can be very slow when Cypress is open (on Windows at least ).
In the Docker environment, it seems to be fine as I had tests finishing without issues in docker headless but timing out on Windows with cypress open.